### PR TITLE
Improve Kerberos auth test stability

### LINKED ISSUE #121: integration tests have started to fail - testKerberosContainer
With the last merge (pr #120) the integration tests have started to fail on CI.  

https://github.com/ozangunalp/kafka-native/actions/runs/7196947318/job/19603932764#step:6:3963

```
Error:  Tests run: 10, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 112.0 s <<< FAILURE! -- in com.ozangunalp.kafka.test.container.KafkaNativeContainerIT
Error:  com.ozangunalp.kafka.test.container.KafkaNativeContainerIT.testKerberosContainer -- Time elapsed: 63.85 s <<< ERROR!
org.testcontainers.containers.ContainerLaunchException: Container startup failed for image quay.io/ogunalp/kafka-native:main-4a8689368ac5241e0ad9ac5f49cbe4ccd1f5c185
```

I'm trying to reproduce the issue in my local environment.

~~I think the container is failing with an issue like this:~~

(edit: removed misleading stack trace)

~~I haven't yet worked out why this system property is failing to be mapped to the bean.~~

### PR DESCRIPTION
**Pull request by ozangunalp**
Change KerberosContainer wait strategy from log to 88 port listen

Remove keytabs created in test-resources, the test now copies them to the target/test-classes/kerberos dir.

Fixes #121

### CODE REVIEW

**Comment by k-wall** [comment]
@ozangunalp lgtm.  thank you for the investigation and producing the fix.

File: kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KerberosContainer.java [hunk 1]
```diff
@@ -36,15 +35,15 @@ public void createTestPrincipals() {
                     "addprinc -randkey client/localhost@EXAMPLE.COM");
             lsResult = execInContainer("kadmin.local", "-q",
                     "ktadd -norandkey -k /client.keytab client/localhost@EXAMPLE.COM");
-            copyFileFromContainer("/kafkabroker.keytab", "src/test/resources/kerberos/kafkabroker.keytab");
-            copyFileFromContainer("/client.keytab", "src/test/resources/kerberos/client.keytab");
+            copyFileFromContainer("/kafkabroker.keytab", "target/test-classes/kerberos/kafkabroker.keytab");
```
Comment (k-wall) [inline kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KerberosContainer.java:38]: just checking my mental model. this part is unrelated to the test instability.  you are just preventing the test run dirtying the source tree.

**Review by k-wall — APPROVED** [review]

File: kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KerberosContainer.java (hunk 1 above)
Comment (ozangunalp) [reply kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KerberosContainer.java:38]: Absolutely
